packages/ember-metal/lib/properties.ts

```diff
@@ -82,13 +94,13 @@ export function INHERITING_GETTER_FUNCTION(name: string) {
     }
   }
 
-  return Object.assign(IGETTER_FUNCTION, {
+  return Object.assign(IGETTER_FUNCTION as InheritingGetterFunction, {
```

Comment:
@krisselden  If I remove the ` as InheritingGetterFunction` then I get:

```
Type '((this: any) => any) & { isInheritingGetter: boolean; }' is not assignable to type 'InheritingGetterFunction'.
  Type '((this: any) => any) & { isInheritingGetter: boolean; }' is not assignable to type '{ isInheritingGetter: true; }'.
    Types of property 'isInheritingGetter' are incompatible.
      Type 'boolean' is not assignable to type 'true'.
```

I'm guessing that's because I defined `InheritingGetterFunction` as `{ isInheritingGetter: true; }` instead of `{ isInheritingGetter: boolean; }`. The former seems like the more correct type though.

I could do this instead:

```
  return Object.assign(IGETTER_FUNCTION, {
    isInheritingGetter: true as true,
  });
```

Reply:
@krisselden Or:

```
return Object.assign(IGETTER_FUNCTION, {
    isInheritingGetter: true,
  }) as InheritingGetterFunction;
```